server/resources/translations/common.es_us.json

```diff
@@ -0,0 +1,18 @@
+{
+  "alertAcknowledged": "Otro personal de auxilio ha confirmado esta solicitud. (No necesita responder a este mensaje).",
+  "alertAdditionalAlert": "Se generó una alerta {{ alertTypeDisplayName }} adicional en {{ deviceDisplayName }}",
+  "alertCompleted": "Gracias. Esta sesión ya está completa. (No necesita responder a este mensaje).",
```

Comment:
According to the translation spreadsheet (https://docs.google.com/spreadsheets/d/1sTUUpZFH4c4isM_pq07xNBjZvoMUtm-B/edit#gid=841616250), I expected there to be exclamation points around the "Gracias" like this: "¡Gracias!"